src/file_resolver.rs

```diff
@@ -184,6 +173,8 @@ impl FileResolver {
     ) -> Result<ResolvedFile, String> {
         let path = PathBuf::from(filename);
 
+        let mut result: Vec<Result<ResolvedFile, String>> = vec![];
+
```

Comment:
Solc fails when an ambiguous import exists. Rather than returning the first resolved import path, we track _all_ possible import paths and fail with an error if there isn't a canonical choice (e.g., not exactly 1).

Reply:
Very nice :+1: